analysis/webservice/algorithms/doms/DomsInitialization.py

```diff
@@ -47,6 +47,10 @@ def init(self, config):
         cassDatacenter = domsconfig.get("cassandra", "local_datacenter")
         cassVersion = int(domsconfig.get("cassandra", "protocol_version"))
         cassPolicy = domsconfig.get("cassandra", "dc_policy")
+        try:
+            cassCreateKeyspaceGranted = domsconfig.get("cassandra", "create_keyspace_granted")
+        except NoOptionError:
```

Comment:
@twnone This line caused errors when running my test, but seemed to work when I changed it to

```suggestion
        except configparser.NoOptionError:
```